client/src/pages/CreateNewPage.css

```diff
@@ -0,0 +1,46 @@
+.Form {
+}
+
+.Page__heading {
+  border: 2px none black;
+  padding: 2em 8em;
+  border-radius: 15px;
+  background: #f4b7b7;
+  display: flex;
+  align-items: center;
+  color: #d84064;
+  margin: 3em;
```

Comment:
the margin is also quite big again. to make your design responsive and work in all shapes and sized you could make the parent element `form` or even `main` a flex container and everything inside is in the middle of the page. :)

```
display: flex;
flex-direction: column;
justify-content: center;
align-items: center;
gap: 2em;
```